Library/Homebrew/cli/named_args.rb

```diff
@@ -327,7 +327,11 @@ def resolve_latest_keg(name)
 
         stable_kegs = kegs.reject { |k| k.version.head? }
 
-        return kegs.max_by { |keg| Tab.for_keg(keg).source_modified_time } if stable_kegs.blank?
+        if stable_kegs.blank?
+          return kegs.max_by do |keg|
+                   [Tab.for_keg(keg).source_modified_time, keg.version.revision]
+                 end
```

Comment:
```suggestion
          return kegs.max_by do |keg|
            [Tab.for_keg(keg).source_modified_time, keg.version.revision]
          end
```

Just a personal preference thing but I'd say this is more consistent with the rest of the codebase. Not a functional difference though so I'd say up to whatever you prefer.

Reply:
Agreed 👍🏻

Reply:
Fine by me. I ran `brew style --fix` on slightly different formatting and it produced the above formatting.